RFC0000-RFC-Process.md

```diff
@@ -35,49 +40,64 @@ All RFC documents shall follow the [RFC template](RFCNNNN-New-RFC-Template.md).
-Note: the Committee may be slower to respond to RFCs where the Author has indicated that they do not plan to implement the RFC.
-
-### Draft-Accepted
-
-The PowerShell Committee has reviewed the RFC and comments, and has voted to accept the RFC as a Draft.
-
-* New comments are not being sought.
-* No one has begun implementing the RFC, and there are no current plans to implement the RFC.
-In this case, the Committee will create `up-for-grabs` Issues in the [PowerShell](https://github.com/PowerShell/PowerShell) repository.
-
-### Experimental
 
-RFCs in the `Experimental` stage have been accepted by the Committee, and code is being written to provide a working example of the proposed design change to get additional feedback.
-
-### Experimental-Accepted
-
-Feedback from the experimental implementation and RFC have been reviewed.
+Note: the Committee may be slower to respond to RFCs where the Author has indicated that they do not plan to implement the RFC.
 
-Because this working prototype already exists in preview builds available on GitHub, the community provide feedback on the implementation as issues in the [PowerShell/PowerShell repository](https://github.com/powershell/powershell)
+### Accepted or Rejected
 
-As the engineering team or code contributor works towards a final implementation, they should submit pull requests to PowerShell-RFC in order to keep the RFC in sync with the implementation.
-These pull requests shall be reviewed and accepted by the Committee, but a formal vote is not necessary.
-The Committee should merely confirm that the changes in the RFC match the working implementation.
+* Feedback from the experimental implementation and RFC has been reviewed.
+* Because this working prototype already exists in preview builds available on GitHub, the community
+  provides feedback on the implementation as issues in the [PowerShell/PowerShell repository](https://github.com/powershell/powershell).
+* As the engineering team or contributor works towards a final implementation,
+  the contributor should submit PRs to PowerShell-RFC in order to keep the RFC in sync with the implementation.
+* The Committee may ask the Author to continue updating the RFC as real world usage of the experimental
+  feature provides new information.
+* When the Committee feels that they have enough information from their own usage of and user
+  feedback on the experimental feature to vote on the RFC,
+  they will vote to accept or reject the RFC.
+  This will be done through a simple merge/rejection of the RFC PR.
 
 ### Rejected
 
-RFCs in the `Rejected` state were rejected by the Committee who decided not to proceed further.
+If the Committee decides not to proceed with the RFC, its PR is simply closed before it is merged.
 
 ### Withdrawn
 
-RFCs in the `Withdrawn` state were rescinded by the RFC Author.
+Similarly, if an Author decides to withdraw their RFC, it will be closed before it is merged.
 
```

Comment:
```suggestion
The Committee should also add the `Withdrawn` label to denote that the Committee
rejected the RFC.
In the future, this can be done automatically with GitHub Actions.

```

Reply:
Same with this one as the Rejected one